### PR TITLE
Refs #30065 - remove now non-existing file from katello.sccs

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
[test katello]

**Comment by mmoll** [comment]
:green_heart:

File: app/assets/stylesheets/katello/katello.scss [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /**
- * = require "katello/jquery.loadmask.css"
```
Comment (jlsherrill) [inline app/assets/stylesheets/katello/katello.scss:2]: i still see this here: https://github.com/Katello/katello/blob/master/vendor/assets/stylesheets/katello/jquery.loadmask.css.scss

Can you explain a bit more?

Comment (mmoll) [reply app/assets/stylesheets/katello/katello.scss:2]: Hrm, that's correct... unfortunately I have no idea thean...

Comment (mmoll) [reply app/assets/stylesheets/katello/katello.scss:2]: Hrm, that's correct... unfortunately I have no idea thean...

**Comment by mmoll** [comment]
I don't have a Katello setup ready for this... https://github.com/theforeman/foreman/pull/7187 needs to get working with Katello.

**Comment by jlsherrill** [comment]
I'll play with it and let you know, maybe its ignoring the vendor directory?

**Comment by jlsherrill** [comment]
https://github.com/rails/sprockets/issues/647  :(

**Comment by jlsherrill** [comment]
looking to see if i can move to use @import

**Comment by jlsherrill** [comment]
i opened up https://github.com/Katello/katello/pull/8771  to hopefully resolve the issue. I also noticed a breakage with the one erb page we have left, so i fixed that (not sure why the tests didn't catch it)

**Pull request closed without merging**